addons/website_event_exhibitor/views/event_exhibitor_templates_list.xml

```diff
@@ -285,7 +285,8 @@
             t-att-data-user-event-manager="is_event_user">
             <a href="#" class="btn btn-primary h3">
                 <t t-if="not is_event_user and not sponsor.event_id.is_ongoing and not event.is_participating">
```

Comment:
Could probably safely remove this in 17.2, unlikely that people customize a complete "t-if" block.